test_lit/ToyFrontend/matmul_mid.toy

```diff
@@ -0,0 +1,13 @@
+# RUN: toy-translate %s -emit=mlir-mid -opt 2>&1 | FileCheck %s
+
+def funcInputs() {
+  var a<3,4> = [0];
+  var b<4,3> = [0];
+  var c<3,3> = [0];
```

Comment:
Would you try to add multiple precision test cases in LIT.
Take
https://github.com/LeiChenIntel/toy-compiler/blob/main/test_lit/ToyFrontend/add_mlir.toy
https://github.com/LeiChenIntel/toy-compiler/blob/main/test_lit/ToyFrontend/add_mlir_mid.toy
as a reference.